packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -255,6 +255,10 @@ export abstract class ActorQueryOperation extends Actor<IActionQueryOperation, I
     }
     return shape.operation.pattern.type === operation.type;
   }
+
+  public static setContextWrapped(context: IActionContext, value: boolean): IActionContext {
```

Comment:
I understand the need for calling this function.
However, it feels error-prone, as forgetting to call it may lead to difficult bugs that are difficult to debug.

An alternative approach could be to make `KEY_CONTEXT_WRAPPED_QUERY_OPERATION` refer not to a boolean, but the actual operation object that is being wrapped. Then you can just check in the actor's test if the wrapped operation is the same as current or not to decide if the test should pass.